sys/net/routing/rpl/rpl_dodag.c

```diff
@@ -31,10 +31,16 @@ char addr_str[IPV6_MAX_ADDR_STR_LEN];
 #endif
 #include "debug.h"
 
+kernel_pid_t rpl_process_pid;
 rpl_instance_t instances[RPL_MAX_INSTANCES];
 rpl_dodag_t dodags[RPL_MAX_DODAGS];
 rpl_parent_t parents[RPL_MAX_PARENTS];
 
+void rpl_trickle_send_dio(void *args) {
+    ipv6_addr_t *mcast = (ipv6_addr_t *) args;
```

Comment:
The RFC states: "When a node receives a unicast DIS without a Solicited Information option, it MUST unicast a DIO to the sender in response." Although, we do not support unicast DIS right now..